### PR TITLE
[zh] sync cluster-upgrade kubelet-credential-provider running-cloud-controller

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 1]
```diff
@@ -116,6 +116,15 @@ kubelet, or upgrade the kubelet on that node and bring the node back into servic
 然后使用一个运行了 kubelet {{< skew currentVersion >}} 版本的新节点替换它；
 或者升级此节点的 kubelet，并使节点恢复服务。
 
+{{< caution >}}
+<!--
+Draining nodes before upgrading kubelet ensures that pods are re-admitted and containers are
+re-created, which may be necessary to resolve some security issues or other important bugs.
+-->
+在升级 kubelet 之前先进行节点排空，这样可以确保 Pod 重新调度并且容器得到重新创建。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:124]:
```suggestion
在升级 kubelet 之前先进行节点排空，这样可以确保 Pod 被重新准入并且容器被重新创建。
```

File: content/zh-cn/docs/tasks/administer-cluster/running-cloud-controller.md [hunk 2]
```diff
@@ -83,17 +83,14 @@ Successfully running cloud-controller-manager requires some changes to your clus
...
+  controller loops in the Kubernetes Controller Manager), then `--cloud-provider=external`
+  must be specified. Otherwise, it should not be specified.
 -->
-* 一定不要为 `kube-apiserver` 和 `kube-controller-manager` 指定 `--cloud-provider` 标志。
-  这将保证它们不会运行任何云服务专用循环逻辑，这将会由云管理控制器运行。未来这个标记将被废弃并去除。
-* `kubelet` 必须使用 `--cloud-provider=external` 运行。
-  这是为了保证让 kubelet 知道在执行任何任务前，它必须被云管理控制器初始化。
+* `kubelet`、`kube-apiserver` 和 `kube-controller-manager` 必须根据用户对外部 CCM 的使用进行设置。
+  如果用户有一个外部的 CCM（不是 Kubernetes 控制器管理器中的 internal cloud controller loops），
+  那么必须添加 `--cloud-provider=external` 参数。否则，不应添加此参数。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/running-cloud-controller.md:94]: 102 行也有变化，请检查。

File: content/zh-cn/docs/tasks/administer-cluster/running-cloud-controller.md [hunk 3]
```diff
@@ -83,17 +83,14 @@ Successfully running cloud-controller-manager requires some changes to your clus
...
+  user's usage of external CCM. If the user has an external CCM (not the internal cloud
+  controller loops in the Kubernetes Controller Manager), then `--cloud-provider=external`
+  must be specified. Otherwise, it should not be specified.
 -->
-* 一定不要为 `kube-apiserver` 和 `kube-controller-manager` 指定 `--cloud-provider` 标志。
-  这将保证它们不会运行任何云服务专用循环逻辑，这将会由云管理控制器运行。未来这个标记将被废弃并去除。
-* `kubelet` 必须使用 `--cloud-provider=external` 运行。
-  这是为了保证让 kubelet 知道在执行任何任务前，它必须被云管理控制器初始化。
+* `kubelet`、`kube-apiserver` 和 `kube-controller-manager` 必须根据用户对外部 CCM 的使用进行设置。
+  如果用户有一个外部的 CCM（不是 Kubernetes 控制器管理器中的 internal cloud controller loops），
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/running-cloud-controller.md:92]:
```suggestion
  如果用户有一个外部的 CCM（不是 Kubernetes 控制器管理器中的内部云控制器回路），
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**